src/v3/pages/reports/meeting_attendance/components/week_box/index.styles.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box, Typography } from '@mui/material';
+import TextField from '@components/textfield';
+import { styled } from '@mui/system';
+
+export const StyledTextField = styled(TextField)({
+  '.MuiInputBase-input': { textAlign: 'center' },
+  '& input[type=number]': {
+    '-moz-appearance': 'textfield',
+  },
+  '& input[type=number]::-webkit-outer-spin-button': {
+    '-webkit-appearance': 'none',
+    margin: 0,
+  },
+  '& input[type=number]::-webkit-inner-spin-button': {
+    '-webkit-appearance': 'none',
+    margin: 0,
+  },
+});
+
+export const StyledTodayTypography = styled(Typography)(({ color }: { color: string }) => ({
```

Comment:
@Logan08691: please make sure that this not produce the Invalid prop to DOM error. Thanks.